frontend/public/style/_layout.scss

```diff
@@ -55,8 +55,8 @@ body,
 
   &__sidebar--hidden {
     position: absolute;
-    right: 23px;
-    top: 1px;
+    right: 13px;
+    top: -1px;
```

Comment:
Now included in #2889, so please drop the changes in this PR.

Reply:
dropped